meta/meta.go

```diff
@@ -117,6 +118,14 @@ func (m *Meta) GenGlobalID() (int64, error) {
 	return m.txn.Inc(mNextGlobalIDKey, 1)
 }
 
+// GenGlobalServerID generates the next ID globally.
+func (m *Meta) GenGlobalServerID() (int64, error) {
```

Comment:
I don't think we need to introduce the global server ID for the TiDB instance because we can use the address (`host:port`) to identify a TiDB instance. We must map ID to address although the TiDB instance has a global instance ID because the ID doesn't contain extra info

Reply:
Ok